tests/size_test.py

```diff
@@ -194,6 +196,10 @@ def testScientificNotation(self):
         self.assertEqual(size.parseSpec("1E-4KB"), Decimal("0.1"))
         self.assertEqual(Size("1E-10KB"), Size(0))
 
+    def testPickling(self):
+        s = Size("10 MiB")
+        self.assertEqual(s, cPickle.loads(cPickle.dumps(s)))
```

Comment:
It's reassuring that the pickling does work, but...

```
c, a = s.__reduce__()
self.assertEqual(s, c(*a))
```

is more properly unit-testy, and you don't have to actually import the pickler that way.